config/tsconfig.base.json

```diff
@@ -1,24 +1,20 @@
 {
   "compileOnSave": false,
   "compilerOptions": {
+    "declaration": true,
     "lib": [
       "es5",
       "es2015",
       "es2015.promise",
       "dom"
     ],
-    "allowJs": true,
     "module": "commonjs",
     "moduleResolution": "node",
-    "noImplicitAny": true,
-    "outDir": "tmp",
-    "target": "ES5",
+    "noImplicitAny": false,
```

Comment:
This is a bummer.

Reply:
Each package has its own `tsconfig.json` so you can set this for your project if you'd like.